### PR TITLE
[US-4162] Removal of base data fixtures

### PR DESCRIPTION
**Pull request by vitek-rostislav**
| Q             | A
| ------------- | ---
|Description, reason for the PR| The concept is just wrong and misused. In base data fixtures, there were supposed to be essential eshop data but often there are demonstrational data, too. Moreover, in production, data fixtures are not loaded so when adding new "base" data, developers are forced to deal with both data fixtures and migrations as well.
|New feature| No 
|BC breaks| No 
|Fixes issues| #237 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -91,7 +91,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - [#228 - show selectbox options](https://github.com/shopsys/shopsys/pull/228)
     - the use of jQuery plugin for selectboxes was modified on Shopsys Framework side so options will now be seen 
 - [#243 - Admin: changed domain icon in e-shop domain administration can be saved](https://github.com/shopsys/shopsys/pull/243)
-    - changed domain icon in e-shop domain administration can be saved 
+    - changed domain icon in e-shop domain administration can be saved
+- copying of localized entities
+    - detection of new locale is now done before multidomain data are created
```
Comment (Miroslav-Stopka) [inline CHANGELOG.md]: If someone asks me, why this change was necessary, I can not answer. Can you explain it to me? Why was it necessary? Why is the previous state no longer correct?

File: packages/framework/src/DataFixtures/Demo/PricingGroupDataFixture.php [hunk 2]
```diff
@@ -19,6 +19,11 @@ class PricingGroupDataFixture extends AbstractReferenceFixture
      */
     private $pricingGroupFacade;
 
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Setting\Setting
+     */
+    private $setting;
```
Comment (Miroslav-Stopka) [inline packages/framework/src/DataFixtures/Demo/PricingGroupDataFixture.php]: Maybe unnecessary?

File: docs/introduction/how-is-the-application-filled-with-data.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# How Is the Application Filled with Data?
```
Comment (Miroslav-Stopka) [inline docs/introduction/how-is-the-application-filled-with-data.md]: Maybe change of header?

File: docs/introduction/how-is-the-application-filled-with-data.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+All basic data that are vital for Shopsys Framework (e.g. administrator, vat, database functions and triggers, etc.) are created in [database migrations](./database-migrations.md).
+
+As the migrations create data for the first domain only,
```
Comment (Miroslav-Stopka) [inline docs/introduction/how-is-the-application-filled-with-data.md]: Please, provide some example.

File: packages/framework/src/DataFixtures/Demo/PricingGroupDataFixture.php (hunk 2 above)
Comment (vitek-rostislav) [reply packages/framework/src/DataFixtures/Demo/PricingGroupDataFixture.php]: yes, fixed :+1:

File: CHANGELOG.md (hunk 1 above)
Comment (vitek-rostislav) [reply CHANGELOG.md]: it is necessary when new locale is added but it is not necessary in this PR so I am removing this change :+1:

Comment (vitek-rostislav) [reply CHANGELOG.md]: it is necessary when new domain with new locale is added :slightly_smiling_face:

File: docs/introduction/how-is-the-application-filled-with-data.md (hunk 3 above)
Comment (vitek-rostislav) [reply docs/introduction/how-is-the-application-filled-with-data.md]: fixed

File: docs/introduction/how-is-the-application-filled-with-data.md (hunk 4 above)
Comment (vitek-rostislav) [reply docs/introduction/how-is-the-application-filled-with-data.md]: examples added

**Pull request merged**